src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQualityUnitTest.java

```diff
@@ -13,6 +13,7 @@
 import org.broadinstitute.hellbender.utils.genotyper.PerReadAlleleLikelihoodMap;
 import org.broadinstitute.hellbender.utils.read.ArtificialReadUtils;
 import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
```

Comment:
Hmm, how is this unit test class passing? Unless I missed something when reading the code (entirely possible, as looking at the diffs vs. GATK3 is slightly disorienting) we are forgetting to take the average in `RMSMappingQuality` when computing the RMS...